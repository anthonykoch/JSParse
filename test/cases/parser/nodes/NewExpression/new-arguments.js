new Hello(123, 'hello')
