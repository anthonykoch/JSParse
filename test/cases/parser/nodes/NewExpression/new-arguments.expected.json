---
name: NewExpression - with args
description: parses new expression with arguments
---

[
  {
    "type": "ExpressionStatement",
    "start": 0,
    "end": 23,
    "expression": {
      "type": "NewExpression",
      "start": 0,
      "end": 23,
      "arguments": [
        {
          "type": "Literal",
          "value": "123",
          "raw": 123,
          "start": 10,
          "end": 13
        },
        {
          "type": "Literal",
          "value": "'hello'",
          "raw": "hello",
          "start": 15,
          "end": 22
        }
      ],
      "callee": {
        "type": "Identifier",
        "name": "Hello",
        "start": 4,
        "end": 9
      }
    }
  }
]
